src/main.h

```diff
@@ -157,8 +157,8 @@ extern CCriticalSection cs_main;
 extern CTxMemPool mempool;
 typedef boost::unordered_map<uint256, CBlockIndex*, BlockHasher> BlockMap;
 extern BlockMap mapBlockIndex;
-extern uint64_t nLastBlockTx;
-extern uint64_t nLastBlockSize;
+extern std::optional<uint64_t> last_block_num_txs;
+extern std::optional<uint64_t> last_block_size;
```

Comment:
Upstream moved these into `miner.h`. We don't have the `BlockAssembler` class, but we could I think still move them there. Non-blocking, this change is not worth re-running CI.